docs/pages/access-controls/sso/okta.mdx

```diff
@@ -88,7 +88,7 @@ Assign groups and people to your SAML app:
 ![Configure APP](../../../img/sso/okta/okta-saml-3.1.png)
 
 Make sure to download the metadata in the form of an XML document. It will be used it to
-configure a Teleport connector:
+configure a Teleport connector. You can find this on the "View Setup Instructions" page under "Optional- Provide the following IDP metadata to your SP provider.":
```

Comment:
Updated screenshot added for review.

Reply:
I moved the image into the correct path for you. There was no error because there was an existing image in that path by the same name no longer in use.